velox/docs/spark_functions.rst

```diff
@@ -4,6 +4,8 @@ Spark Functions
 
 The semantics of Spark functions match Spark 3.5 with ANSI OFF.
 
+Spark functions can be registered by :doc:`struct SparkRegistrationConfig <functions/spark/config>`.
```

Comment:
Doc suggestion:
Spark functions can be registered with :doc:\`user specified configurations <functions/spark/config>\`.